metrics/graphite.go

```diff
@@ -60,7 +60,7 @@ func buildUniquePrefix(basePrefix string) string {
 	if err != nil {
 		log.Fatalf("Unable to get hostname for metrics key: %s", err)
 	}
-	return fmt.Sprintf("%s.%s.%s", basePrefix, normalizeValue(hostname), metricsID)
+	return fmt.Sprintf("%s.%s.%s", basePrefix, normalizeValue(hostname), normalizeValue(metricsID))
```

Comment:
I think that this value should be normalised earlier as it is printed to logs.

Reply:
Hmmm, may be true. Bu this way we can actually test it, plus all the parameters are normalized at the last possible moment (which I consider good). I'll find where we print this stuff.

Reply:
Just one log at startup, normalizing twice should not be a performance problem. ;)

Reply:
I'd prefer to normalize onece at the top 

```go
metricsID = normalizeValue(id)
```

Reply:
That is an optimization, but is also very difficult to test. I'd rather have it proven to work.